app/lib/shared/versions.dart

```diff
@@ -7,18 +7,18 @@ import 'package:pub_semver/pub_semver.dart';
 import 'utils.dart' show isNewer;
 
 // update this whenever one of the other versions change
-final String runtimeVersion = '2018.09.03';
+final String runtimeVersion = '2018.09.05';
 final Version semanticRuntimeVersion = new Version.parse(runtimeVersion);
 
 // keep in-sync with SDK version in .travis.yml, .mono_repo.yml and Dockerfile
 final String runtimeSdkVersion = '2.1.0-dev.2.0';
 final String toolEnvSdkVersion = '2.0.0';
 
 // keep in-sync with app/pubspec.yaml
-final String panaVersion = '0.12.1';
+final String panaVersion = '0.12.2';
 final Version semanticPanaVersion = new Version.parse(panaVersion);
 
-final String flutterVersion = '0.6.2';
+final String flutterVersion = '0.7.5';
```

Comment:
Has this flutter been tested?

Reply:
Not yet, I'll do it later today in staging.

Reply:
Looks good in staging.